source/edge-server/configure.txt

```diff
@@ -134,271 +116,160 @@ EC2, you must open ports to accept incoming connections:
-               {
-                 "publicKeyPath": "YOUR_PUBLIC_KEY_PATH/cert.pem",
-                 "privateKeyPath": "YOUR_PRIVATE_KEY_PATH/certkey.pem"
-               }
-             ]
-           }
-         }
+      You can provide the following optional flags to specify additional
+      configuration details:
 
       .. list-table::
          :header-rows: 1
+         :widths: 25 10 65
 
          * - Field name
+           - Type
            - Value
 
-         * - ``clientAppId``
-           - The unique ID of your App Services App. For information about 
-             finding the App ID, refer to :ref:`find-app-id`.
-
-         * - ``query``
-           - The ``query`` field defaults to ``"*"``, which automatically 
-             syncs all data in all of the collections in your :ref:`App 
-             Services schema <schemas>`.
-           
-             If you would like to filter the data that syncs to your Edge 
-             Server, pass an object containing one or more collections 
-             and queries to sync a subset of your data with Edge Server.
-             This may resemble:
-
-             .. code-block:: json
-
-                {
-                  "clientAppId": "sync-template-app-zkiee",
-                  "query": {
-                    "YOUR-COLLECTION-NAME": "YOUR-QUERY",
-                    "YOUR-COLLECTION-NAME": "YOUR-QUERY"
-                  },
-                  ... additional configuration fields...
-                }
-
-         * - ``YOUR-COLLECTION-NAME``
-           - If you're passing a query object to filter the data that syncs
-             to the Edge Server, this is the name of a collection in your 
-             database. If you're using a template app, this is the ``Item`` 
-             collection. Your Edge Server query may contain many collections.
-            
-         * - ``YOUR-QUERY``
-           - If you're passing a query object to filter the data that syncs
-             to the Edge Server, this is the query that determines what 
-             documents the Edge Server can sync with a collection. You can 
-             use :ref:`Realm Query Language <rql>` to define this query, 
-             with the same caveats around :ref:`Flexible Sync RQL 
-             Limitations <flexible-sync-rql-limitations>`.
-             
-             For example, ``truepredicate`` means the Edge Server syncs 
-             all documents with the collection. You might only want to sync 
-             a subset of documents with an Edge Server, so you could pass a
-             RQL query similar to ``“store_id == 42”``.
-
-         * - ``cloudSyncServerAuthSecret``
-           - The authorization secret that your Product or Account 
-             Representative provided when you contacted them to get 
-             Edge Server enabled for your App.
-
-         * - ``httpListenPort``
-           - The port on which Edge Server listens for HTTP connections. The
+         * - ``--config-path``
```

Comment:
I believe Sudarshan talked to you about this but this file will change soon so that it can only be used to set the query for the config.